### PR TITLE
Simple payments: nudges

### PR DESCRIPTION
**Pull request by artpi**
This PR blocks simple-payments for any plan less than premium and puts delightful nudge:

zrzut ekranu 2017-08-03 o 17 26 32

It also cleans up feature availability in /plans : 
zrzut ekranu 2017-08-03 o 17 37 18

## Testing

For both JP and wpcom:
- free and personal should get you nudge
- premium and business should gnt you feature

### CODE REVIEW

File: client/components/tinymce/plugins/simple-payments/dialog/index.jsx [hunk 1]
```diff
@@ -333,8 +336,44 @@ class SimplePaymentsDialog extends Component {
...
+				isVisible={ showDialog }
+				onClose={ onClose }
+				buttons={ [
+					<Button onClick={ onClose }>
+						{ translate( 'Cancel' ) }
+					</Button>,
+				] }
+				additionalClassNames="editor-simple-payments-modal"
+			>
+				<Navigation
```
Comment (jsnajdr) [inline client/components/tinymce/plugins/simple-payments/dialog/index.jsx]: Do we need to render the navigation at all? It's quite useless on this dialog.

File: client/components/tinymce/plugins/simple-payments/dialog/index.jsx [hunk 2]
```diff
@@ -333,8 +336,44 @@ class SimplePaymentsDialog extends Component {
...
 
+	renderEmptyDialog( content ) {
+		const { onClose, translate, showDialog } = this.props;
+		return (
+			<Dialog
+				isVisible={ showDialog }
+				onClose={ onClose }
+				buttons={ [
+					<Button onClick={ onClose }>
+						{ translate( 'Cancel' ) }
```
Comment (jsnajdr) [inline client/components/tinymce/plugins/simple-payments/dialog/index.jsx]: nit: "Close" might be a little bit better than "Cancel"

File: client/components/tinymce/plugins/simple-payments/dialog/index.jsx [hunk 3]
```diff
@@ -333,8 +336,44 @@ class SimplePaymentsDialog extends Component {
...
+					activeTab={ 'list' }
+					paymentButtons={ [] }
+					onChangeTabs={ noop }
+				/>
+				{ content }
+			</Dialog>
+		);
+	}
+
+	returnTrue() {
```
Comment (jsnajdr) [inline client/components/tinymce/plugins/simple-payments/dialog/index.jsx]: Very funny method :) `UpgradeNudge` should be changed to allow both function and boolean as the value of `shouldDisplay`.

**Comment by iamtakashi** [comment]
This looks strange :) This really should use the empty content component. See the difference.

image

image

**Comment by donpark** [comment]
@iamtakashi When I use empty content component, result looks very different. Graphics is I assume the default.

screenshot_307

**Comment by iamtakashi** [comment]
@donpark Thanks for taking this. That looks like a good start, could you make a new PR? Let's polish it there.

**Pull request merged**